Cargo.toml

```diff
@@ -16,7 +16,8 @@ travis-ci = { repository = "async-std/async-tls" }
 appveyor = { repository = "async-std/async-tls" }
 
 [dependencies]
-futures = "0.3.4"
+futures-io = "0.3.5"
+futures-util = "0.3.5"
```

Comment:
This can be replaced with `futures-core` because there's `futures_core::ready!`

Reply:
Done.